assets/app/view/game/game_meta.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module View
+  class GameMeta < Snabberb::Component
+    needs :game
+
+    def render
+      children = [h(:h3, 'Game Info')]
+
+      if (publisher = @game.class::GAME_PUBLISHER)
+        children << h(:p, [
+            'Published by ',
+            *Lib::Publisher.link_list(component: self, publishers: Array(publisher)),
+          ])
+      end
+      children << h(:p, "Designed by #{@game.class::GAME_DESIGNER}") if @game.class::GAME_DESIGNER
+      children << h(:p, "Implemented by #{@game.class::GAME_IMPLEMENTER}") if @game.class::GAME_IMPLEMENTER
+      if @game.class::GAME_RULES_URL.is_a?(Hash)
+        @game.class::GAME_RULES_URL.each do |desc, url|
+          children << h(:p, [h(:a, { attrs: { href: url, target: '_blank' } }, desc)])
+        end
+      else
+        children << h(:p, [h(:a, { attrs: { href: @game.class::GAME_RULES_URL, target: '_blank' } }, 'Rules')])
+      end
+      if @game.optional_rules.any?
```

Comment:
empty

Reply:
This was pre-existing code, but I can change it.